### ISSUE TITLE
Question on how to get comdump to run

### ISSUE THREAD

**Issue by asciim0** [richardlehane/comdump#1]
Hey Richard,

I'm trying to get comdump to run. Since I couldn't find the binaries, I'm trying to get it to work with Go ... but unfortunately I'm not very familiar with Go (my main success has been getting pdfcpu to run so far). So, after cloning the github comdump repo, I tried "go install", but get:

go: inconsistent vendoring in C:\Go\src\comdump:
    go.mod required github.com/richardlehane/comdump  but vendor/modules.txt does not include it.
     run 'go mod tidy; go mod vendor' to sync

and when running go mod tidy; go mod vendor and after that go install, it still doesn't work. 

Do you have any pointers how I can get it to work? 

Thanks in advance!
Micky

**Comment by richardlehane** [comment]
Hi Micky
it looks like you've encountered this issue: https://github.com/golang/go/issues/34657
I.e. rather than clone to the c:\Go\src directory (which is intended for source that is part of the golang standard library), clone anywhere else on your computer e.g. c:\coolstuff\comdump
Also, in debugging this, I saw that comdump was using an older version of my mscfb library so I updated the comdump dependencies and it might be worth cloning again to get that update. 
Hope this helps and thanks for trying this!
Richard
